appserver/packager/appserver-core/pom.xml

```diff
@@ -263,7 +263,7 @@
 <!--        </dependency>-->
         <dependency>
             <groupId>fish.payara.server.internal.packager</groupId>
-            <artifactId>jakarta.servlet-api-transition</artifactId>
+            <artifactId>jakarta.servlet-api</artifactId>
             <version>${project.version}</version>
```

Comment:
why do we move back to original api, but other references i. e. admingui were not updated?